README.md

```diff
@@ -1,4 +1,7 @@
-# `egui_dock`: docking support for `egui`
+[![Crates.io](https://img.shields.io/crates/v/egui_dock)](https://crates.io/crates/egui_dock)
+[![docs.rs](https://img.shields.io/docsrs/egui_dock)](https://docs.rs/egui_dock/)
+
+# `egui_dock`: docking support for [egui](https://github.com/emilk/egui)
 
 Credit goes to [@Iain-dono](https://github.com/lain-dono) for implementing the actual library.
```

Comment:
According to the MD rule [041](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md041):
> First line in a file should be a top-level heading

Change it to:
```diff
+ # `egui_dock`: docking support for [egui](https://github.com/emilk/egui)
+
+ [![Crates.io](https://img.shields.io/crates/v/egui_dock)](https://crates.io/crates/egui_dock)
+ [![docs.rs](https://img.shields.io/docsrs/egui_dock)](https://docs.rs/egui_dock/)
-
- # `egui_dock`: docking support for [egui](https://github.com/emilk/egui)
```